fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -89,6 +89,17 @@ object ItemKicker {
     case FreeHtmlKickerWithLink(body, _) => Some(body)
     case _ => None
   }
+
+  /** Return the canonical display text for a kicker */
+  def kickerDisplayText(itemKicker: ItemKicker): Option[String] = itemKicker match {
```

Comment:
If the answer to the last one is “no, let's leave it as is”, then perhaps `kickerHTML` is not the best name for it; `kickerContents` may be a better name (it returns the contents of kickers which have them, in text or HTML, and `None` for contentless kickers).